app/concepts/todo/create.rb

```diff
@@ -1,50 +1,28 @@
 class Todo::Create < Trailblazer::Operation
+  # i put this because when we enter a nil or blank title form i got error undefine title
+  include Model
+  model Todo
+  #builds do |params|
+    #return invalid! if params[:todo][:title].blank?
+  #end
+
+  contract do
+    property :title, validates: {presence: true}
+    #validates :title, presence: true
+  end
 
-  # Logic for creating a TODO
```

Comment:
Good to see some deletions here. Our senior dev don't want comments polluting the code. Can we get rid of the comments?